cli/ops.rs

```diff
@@ -142,6 +142,13 @@ pub fn dispatch_all(
   (base.sync(), boxed_op)
 }
 
+pub fn op_selector_compiler(inner_type: msg::Any) -> Option<OpCreator> {
+  match inner_type {
+    msg::Any::FetchModuleMetaData => Some(op_fetch_module_meta_data),
+    _ => op_selector_std(inner_type),
```

Comment:
Compiler only does FetchModuleMetaData, right? I think you can do `unreachable!()` here.

Reply:
We also need WorkerGetMessage and WorkerPostMessage

Reply:
Oh right - so add those but then branch to unreachable

Reply:
also start